module-services/service-bluetooth/ServiceBluetooth.cpp

```diff
@@ -348,13 +351,25 @@ auto ServiceBluetooth::handle(message::bluetooth::DisconnectResult *msg) -> std:
     return sys::MessageNone{};
 }
 
-auto ServiceBluetooth::handle(message::bluetooth::ResponsePasskey *msg) -> std::shared_ptr<sys::Message>
+auto ServiceBluetooth::handle(message::bluetooth::ResponseAuthenticatePin *msg) -> std::shared_ptr<sys::Message>
+{
+    /// TODO to be added in next PRs
```

Comment:
```suggestion
    /// TODO to be added in next PRs
        auto passKey = msg->getPasskey();
    bluetooth::GAP::respondPinCode(passKey, msg->getDevice());
    return sys::MessageNone{};
```

At this moment responding the pin/passkey is done in the same place, as the logic is driven by flag - it will be changed

Reply:
ok